### PR TITLE
Authorize.net: certificates added for digicert

### PR DESCRIPTION
**Pull request by dawoodmalhi**
Added new certificates since authorize.net is migrating from Entrust to DigiCert.

**Reference Article:**  https://support.authorize.net/knowledgebase/Knowledgearticle/?code=KA-05545
**Certificates Source:** https://support.authorize.net/knowledgebase/Knowledgearticle/?code=000003009

### CODE REVIEW

**Comment by dawoodmalhi** [comment]
@yunnydang can you please review these changes?

File: lib/certs/cacert.pem [hunk 1]
```diff
@@ -3212,3 +3212,61 @@ gLWi5h+xEk8blTAKBggqhkjOPQQDAwNoADBlAjEA31SQ7Zvvi5QCkxeCmb6zniz2C5GMn0oUsfZk
 vLtoURMMA/cVi4RguYv/Uo7njLwcAjA8+RHUjE7AwWHCFUyqqx0LMV87HOIAl0Qx5v5zli/altP+
 CAezNIm8BZ/3Hobui3A=
 -----END CERTIFICATE-----
+
+DigiCert Global Root G2
```
Comment (bradbroge) [inline lib/certs/cacert.pem:3216]: this already exists in the file (ln 1759)

File: lib/certs/cacert.pem [hunk 2]
```diff
@@ -3212,3 +3212,61 @@ gLWi5h+xEk8blTAKBggqhkjOPQQDAwNoADBlAjEA31SQ7Zvvi5QCkxeCmb6zniz2C5GMn0oUsfZk
...
+5uNu5g/6+rkS7QYXjzkwDQYJKoZIhvcNAQELBQADggEBAGBnKJRvDkhj6zHd6mcY
+1Yl9PMWLSn/pvtsrF9+wX3N3KjITOYFnQoQj8kVnNeyIv/iPsGEMNKSuIEyExtv4
+NeF22d+mQrvHRAiGfzZ0JFrabA0UWTW98kndth/Jsw1HKj2ZL7tcu7XUIOGZX1NG
+Fdtom/DzMNU+MeKNhJ7jitralj41E6Vf8PlwUHBHQRFXGU7Aj64GxJUTFy8bJZ91
+8rGOmaFvE7FBcf6IKshPECBV1/MUReXgRPTqh5Uykw7+U0b6LJ3/iyK5S9kJRaTe
+pLiaWN0bfVKfjllDiIGknibVb63dDcY3fe0Dkhvld1927jyNxF1WW6LZZm6zNTfl
+MrY=
+-----END CERTIFICATE-----
+
+DigiCert EVRSACA G2
```
Comment (bradbroge) [inline lib/certs/cacert.pem:3241]: I'm unable to find any public record of this cert.  Can you provide any documentation for it?

**Review by bradbroge — CHANGES_REQUESTED** [review]
1 cert already exists in the file, and I'm unclear where the other came from.  I'm inclined to deny this change.

File: lib/certs/cacert.pem (hunk 2 above)
Comment (dawoodmalhi) [reply lib/certs/cacert.pem]: Authorize.net has provided these certificates as attached on a KnowledgeBase article. I have attached the respective link in the PR Comment. But following is the direct link for download.
https://support.authorize.net/_entity/annotation/c98db1ed-e281-ef11-ac21-6045bd065e37

This is also available on official page of DigiCert.  
https://www.digicert.com/kb/digicert-root-certificates.htm#:~:text=DigiCert%20EV%20RSA%20CA%20G2

File: lib/certs/cacert.pem (hunk 1 above)
Comment (dawoodmalhi) [reply lib/certs/cacert.pem:3216]: I guess that is a different certificate. I don't know the usecase of the old one. But this was Provided by Authorize.net on their documentation page. Link is attached in the PR comment.

I downloaded this certificate from the official DigiCert website. Following is the link to the respective page.
https://www.digicert.com/kb/digicert-root-certificates.htm#:~:text=expired%20%C2%A0%C2%A0revoked-,DigiCert%20Global%20Root%20G2,-Download%20PEM%20%7C

Comment (Ligator) [reply lib/certs/cacert.pem:3216]: @dawoodmalhi, please remove this repeated certificate so bradbroge can reconsider merging this PR with the DigiCert certificate only.

Comment (dawoodmalhi) [reply lib/certs/cacert.pem]: this certificate has been removed from the PR.

**Review by bradbroge — COMMENTED** [review]
We will plan to merge this either late this week, or early next week. I see that there is a deadline of 23/24th October, we will merge before then.

**Review by bradbroge — APPROVED** [review]

**Comment by skwidz** [comment]
@bradbroge Is the plan still to merge this PR before the 23rd?

**Comment by bradbroge** [comment]
This will be merged tomorrow.

**Comment by bradbroge** [comment]
nevermind, I can deploy it in the next hour or two 👍🏻

**Comment by diogogallo** [comment]
Hey @dawoodmalhi and @bradbroge , thank you for updating the certificates and getting this merged. Any plans to cut a new version soon?

**Pull request merged**